tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "After the end of the production run, it would be useful to obtain some properties that will give us an insight into our protein system.\\\n",
+    "There are various so-called ensembles that are used for protein simulations - probably the most common is a system where the number of particles, the pressure and the temperature are held constant (NPT). This is usually achieved by means of a heat-bath. Nevertheless, it is usually a good idea to check these as a function of time through the trajectory just to make sure nothing unexpected happened. First let us check the temperature of our simulation.\n",
+    "\n",
+    "`% gmx energy -f md.edr -s md.tpr -o 1hsg_temperature.xvg`\n",
+    "\n",
+    "The program will then present you with a large table of all the values recorded in the energy (.edr) file. We want to examine temperature so type 13, press enter and then 0 and press enter again. The program will then analyse the temperature and present some statistics of the analysis at the end.\n",
+    "\n",
+    "Another set of properties that is quite useful to examine is the various energetic contributions to the energy. The total energy should be constant. but the various contributions can change and this can sometimes indicate something interesting or strange happening in your simulation. Let us look at some energetic properties of the simulation.\n",
+    "\n",
+    "`% gmx energy -s md.tpr -f md.edr -o 1hsg_energies.xvg`\n",
+    "\n",
+    "We shall select short-range lennard-jones (7), short range coulombic (9) and the potential energy (11). End your selection with a zero.\n",
+    "\n",
+    "Finally, we need to renumber the em.gro file so that the residues of the two chains will not have the same residue number. This will be necessary for when we calculate the RSF values.\n",
+    "\n",
+    "`% gmx editconf -f em.gro -o em_renumbered.gro -resnr 1`\n",
+    "\n",
+    "We will plot and explore the temperature and the energetic components that we obtained in the next section of the tutorial.\n",
+    "\n",
+    "\n",
+    "### File Transfer \n",
+    "\n",
+    "As soon as the simulation is finished, you should go to your local terminal and transfer the files from the remote directory to your local directory.\\\n",
+    "First, in your local terminal, go to the `OxCompBio/tutorials/MD directory`.\n",
+    "\n",
+    "The use `scp` to transfer the remote run subdirectory to your local directory:\n",
+    "\n",
+    "`% scp username@oscgate.arc.ox.ac.uk:/home/username/run/ .`\n",
```

Comment:
We probably need to be clearer as to where they are copying to (i.e. the place where the notebooks exist)?

Reply:
Clarified it.